src/Core/AbstractConnection.php

```diff
@@ -83,6 +84,7 @@ public function __construct(MessageProcessor $messageProcessor, LoopInterface $l
         $this->handshake = $handshake;
         $this->messageProcessor = $messageProcessor;
         $this->loop = $loop;
+        $this->logger = new NullLogger();

```

Comment:
heu ?

Reply:
wut ?

Reply:
I really think an attribute named "logger" should never contain "null" but "NullLogger". There are many similar other points that may be improved in that class. But well, here is one.